plugins/smartling/src/smartling.ts

```diff
@@ -11,125 +11,132 @@ export type Project = {
 };
 
 export class SmartlingApi {
-  private privateKey?: string;
-  loaded?: Promise<void>;
-  resolveLoaded?: () => void;
-  // TODO: basic cache
-  getBaseUrl(path: string, search = {}) {
-    const params = new URLSearchParams({
-      ...search,
-      pluginId: pkg.name,
-      apiKey: appState.user.apiKey,
-    });
+         private privateKey?: string;
```

Comment:
seems extra indentation snuck in for the whole file?

Reply:
yes, but ther is one small addition of `isPluginPrivateKeySame` method. 
I tried pre-pr and linting explicitly, but the extra indendation always applied back.